File: app/src/main/java/com/aircraft/toolmanagment/data/entity/ToolTransfer.java
package com.aircraft.toolmanagment.data.entity;

import androidx.room.Entity;
import androidx.room.PrimaryKey;

@Entity(tableName = "tool_transfers")
public class ToolTransfer {
    @PrimaryKey(autoGenerate = true)
    public int id;
    public int toolId;
    public int oldBorrowerId;
    public int newBorrowerId;
    public long transferTime;
    public String transferReason;
    public String location;

    public ToolTransfer() {}
}

File: app/src/main/java/com/aircraft/toolmanagment/data/entity/ToolTransfer.kt
package com.aircraft.toolmanagment.data.entity

import androidx.room.Entity
import androidx.room.PrimaryKey

@Entity(tableName = "tool_transfers")
data class ToolTransfer(
    @PrimaryKey(autoGenerate = true)
    val id: Int = 0,
    val toolId: Int = 0,
    val oldBorrowerId: Int = 0,
    val newBorrowerId: Int = 0,
    val transferTime: Long = 0,
    val transferReason: String = "",
    val location: String = ""
)